app/helpers/shifts_helper.rb

```diff
@@ -1,15 +1,15 @@
 module ShiftsHelper
 
   def shift_vote_collection_dates(polls)
-    date_options((start_date(polls)..end_date(polls)))
+    date_options((start_date(polls)..end_date(polls)), 0)
```

Comment:
Instead of using a harcoded `0` here (or `1` at line 8)  I would give more context to those values as it could be very hard to understand for someone that doesn't know about Poll Shift tasks being a enumeration.

Maybe since:
```
irb(main):002:0> Poll::Shift.tasks
=> {"vote_collection"=>0, "recount_scrutiny"=>1}
```

Using ` Poll::Shift.tasks[:vote_collection]` and `Poll::Shift.tasks[:recount_scrutiny]` could be way better to make code easier to read.